SurveyService/src/test/java/com/revature/servicetest/AuthServiceJWTTest.java

```diff
@@ -0,0 +1,94 @@
+package com.revature.servicetest;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.util.Date;
+
+import javax.xml.bind.DatatypeConverter;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import com.revature.service.AuthService;
+import com.revature.util.InvalidBatchIdException;
+import com.revature.util.InvalidSurveyIdException;
+
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.MalformedJwtException;
+
```

Comment:
Rewrite tests, make sure they are testing what is actually being validated in this service